sources/Inputs/include/Configuration.h

```diff
@@ -98,6 +98,42 @@ class Configuration {
     return dsoVoltageLevel_;
   }
 
+  /**
+   * @brief Get the Time at which the simulation will start
+   *
+   * @returns the parameter value
```

Comment:
return the start time value, similar below

Reply:
Done